create_embedding.py

```diff
@@ -0,0 +1,64 @@
+import nltk
+from gensim.models import Word2Vec
+from tqdm import tqdm
+import pandas as pd
+import numpy as np
+
+
+def invalid_sentence_vec(v_size):
+    return np.zeros(v_size, dtype=np.float32)
+
+
+def make_sentence_embeddings():
+    train = pd.read_csv('./data/silver/train.csv')
+    valid = pd.read_csv('./data/silver/valid.csv')
+    test = pd.read_csv('./data/silver/test.csv')
+
+    corpus = []
+    for df, name in [(train, "train"), (valid, "valid")]:
+        for row in tqdm(df.itertuples(), total=len(df), desc=name):
+            corpus.append(nltk.word_tokenize(row.text))
+
+    v_size = 100  # 100 is default
+    w2v_model = Word2Vec(corpus, min_count=1, vector_size=v_size)
```

Comment:
One last thing, it would be nice if the model is saved so we can use it during deployment

Reply:
Or use the pretrained ones... which I thought we will do initially ... 

I usually use it like this, it downloads the pretrained one from the internet and then uses it
```
import gensim.downloader as w2v_api
w2v_model = w2v_api.load("word2vec-google-news-300")
```

https://radimrehurek.com/gensim/models/word2vec.html#pretrained-models

Reply:
Or if we actually want to train it (maybe it gives better results idk), then it better be extracted into a separate place and then have an option to train a new one or use the previous one. Because training a new one will basically make the previous models and results in general obsolete.